pyrk/ver.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd say we should also delete the original version.py file. (You can use git mv for this or git rm).